app/controllers/bulk_upload_sales_logs_controller.rb

```diff
@@ -28,7 +28,7 @@ def current_year
   end
 
   def in_crossover_period?
-    FormHandler.instance.forms.values.any?(&:in_crossover_period?)
+    FormHandler.instance.in_crossover_period?
```

Comment:
Would it make sense to only check the lettings forms crossover?

I suspect the crossover dates for lettings and sales would be the same, but in case for some reason they aren't